### PR TITLE
add lprnet

### PR DESCRIPTION
**Pull request by hengyang123**
name,shape,gops,time(ms),mac_utilization,cpu_usage,ddr_utilization
lprnet_bm1684x_int8_asym,1x3x24x94,30.960,0.443,218.28%,5.00%,14.75%
lprnet_bm1684x_f32,1x3x24x94,30.960,0.729,2124.36%,5.00%,27.43%
lprnet_bm1684x_int8_sym,1x3x24x94,30.960,0.440,219.94%,5.00%,14.51%

SOPHGO Toolchain v0.8.299-g9e37af59-20230307
libsophon 0.4.5

### CODE REVIEW

File: vision/detection/lprnet/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Model
+
+|Model                      |Size                                                                                                                      |
+|:--------------------------|:-------------------------------------------------------------------------------------------------------------------------|
+|lprnet                     |[1.7M](https://github.com/xuexingyu24/License_Plate_Detection_Pytorch/blob/master/LPRNet/weights/Final_LPRNet_model.pth)  |
+
+## Dataset
+
+[COCO2017](https://cocodataset.org/)
```
Comment (Jeffwhen) [inline vision/detection/lprnet/README.md:17]: 数据集不对？

**Review by Jeffwhen — CHANGES_REQUESTED** [review]

File: vision/detection/lprnet/README.md (hunk 1 above)
Comment (hengyang123) [reply vision/detection/lprnet/README.md:17]: 贴的是cali用的数据集，

**Comment by XiangBin-gh** [comment]
合并成一笔提交

**Comment by hengyang123** [comment]
已合并

**Pull request closed without merging**